SMSGlobal/OTP.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using SMSGlobal.Response;
+using System.Configuration;
+using System.Net.Http;
+
+
+namespace SMSGlobal.api
+{
+    public class OTP
+    {
+        public Credentials Credentials { get; set; }
+        public OTP(Credentials creds)
+        {
+            Credentials = creds;
+        }
+
+        public async System.Threading.Tasks.Task<OTPRespone> OTPSend(Object payload)
```

Comment:
Not sure how this payload object works. Can we define the structure for it explicitly?

Reply:
We are passing payload when running the test cases, the reason for leaving it as it is so that we can pass different payloads for different requests.